go-controller/pkg/ovn/ovn.go

```diff
@@ -345,10 +345,7 @@ type emptyLBBackendEvent struct {
 }
 
 func newModeEgressIP() modeEgressIP {
-	if config.Gateway.Mode == config.GatewayModeLocal {
-		return &egressIPLocal{}
-	}
-	return &egressIPShared{}
+	return &egressIPMode{}
```

Comment:
"mode" here is an artifact of the old architecture and suggests there's more cleanup to be done (perhaps in a later PR)

Reply:
Exactly, as mentioned here: https://github.com/ovn-org/ovn-kubernetes/pull/1677#issuecomment-697045112 I didn't want to make this PR too big. There will be a follow up PR